duckduckgo/Android
# Add newer WebView cache directory to deletion whitelist

Task/Issue URL: https://app.asana.com/0/608920331025315/1154972129856323
Tech Design URL: 
CC: 

**Description**:
Adds a new whitelisted directory to our exclusions to match the newer Chrome cache directory.

**Steps to test this PR**:
1. Using the latest stable Chrome version, visit a few different sites
1. Perform a 🔥data clear
1. Inspect the data held on disk, and ensure it doesn't contain any previous browsing session


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
